src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -27,8 +27,8 @@ public static class TreeExtensions
         /// <param name="learningRate">The learning rate.</param>
         /// <param name="advancedSettings">Algorithm advanced settings.</param>
         public static FastTreeRegressionTrainer FastTree(this RegressionContext.RegressionTrainers ctx,
-            string label = DefaultColumnNames.Label,
-            string features = DefaultColumnNames.Features,
+            string label,
```

Comment:
I have seen in other files that features comes before label. We should pick one order and stick to it, otherwise it is very confusing. I would suggest features, label, optional weight. But either ways could work!

Reply:
Actually I think you already made the decision, label, features, optional weight, which is consistent with all other files in this PR. Maybe we should consider opening an issue and updating the order these in another PR?

---
In reply to: [230870480](https://github.com/dotnet/machinelearning/pull/1524#discussion_r230870480) [](ancestors = 230870480)